js/components/visualizations/filter-panel/filter-panel.js

```diff
@@ -0,0 +1,22 @@
+define(
+  [
+    'knockout',
+    'text!components/visualizations/filter-panel/filter-panel.html',
```

Comment:
should this use relative paths (text!./filter-panel.html)?

Also, does the less plugin support relative paths?  It should...